Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs

```diff
@@ -104,11 +105,27 @@ public void UpdateController(InteractionSourceState interactionSourceState)
         private void UpdateControllerData(InteractionSourceState interactionSourceState)
         {
             LastSourceStateReading = interactionSourceState;
+            var lastState = TrackingState;
 
-            isControllerTracked = interactionSourceState.sourcePose.TryGetPosition(out currentControllerPosition);
-            ControllerState = isControllerTracked ? ControllerState.Tracked : ControllerState.NotTracked;
+            switch (interactionSourceState.sourcePose.positionAccuracy)
+            {
+                case InteractionSourcePositionAccuracy.None:
+                    TrackingState = TrackingState.NotTracked;
+                    break;
+                case InteractionSourcePositionAccuracy.Approximate:
```

Comment:
Recommend removing as #2347 handles this and we anticipate there may be a platform that has approximate orientation as well (which would break the hierarchy implied here).